.github/workflows/check_links.yml

```diff
@@ -0,0 +1,71 @@
+name: Check Links
+
+on:
+  workflow_dispatch:
+# Disable runs on PR's to avoid getting rate limited because of too many requests
+#  pull_request:
+#    branches:
+#      - master
+  pull_request_review:
+    types: [submitted]
+  push:
+    branches:
+      - master
+  schedule:
+    # Runs at the start of each month (UTC)
+    - cron: '0 0 1 * *'
+
+jobs:
+  check_urls:
+    # Do not run the scheduled workflow on forks + allow manual runs on 🕵️ keyword in PR reviews
+    if: >
+      ( github.event_name != 'schedule' ||
+        github.repository_owner == 'Cockatrice'
+        ) ||
+      ( github.event_name == 'pull_request_review' &&
+        ( github.event.review.author_association == 'OWNER' ||
+          github.event.review.author_association == 'MEMBER'
+        ) &&
+        contains( github.event.review.body, '@github-actions :detective:' )
+      )
```

Comment:
Did I got the conditions right @ebbit1q ?

Reply:
why does it care about reviews?

Reply:
That's by choice.
It could be on comments, but again, it would then trigger on every single comment in every issue and every PR... that's not useful. I don't see a downside in doing a more narrow check and keeping the action history more clean and relevant? 🤷 

It's also fitting. It can be part of the review process to inspect links and get reports on their health.

Reply:
I don't understand, this action could simply run when pushed, that's when the file is changed

the reason why we don't run the update date action on push but on review is because it directly writes to the branch (though I am considering simply having it create extra commits on master when pushed on master instead of adding a bot account to the org as a workaround for pull requests from forks but I don't have the permissions to create that)

Reply:
>this action could simply run when pushed, that's when the file is changed

That's how this PR started... on push to master and PRs.

- **push to master** (+ PR merge)
  We get the important information too late then and fixes need a new, second PR
- **PRs**
  That's when anybody creating a PR and committing frequent updates could get us temporarily blocked (happened with the current lychee tool already)

Your approach sounded appealing to help with this as well.

Reply:
The maintainer of the lychee link tool and the related action has a "only check changed links in PRs" feature on his list.
That would be a great fit and can be combined to run full checks on a regular schedule while checking changes only instantly in PRs to minimize the amount of queries and prevent running into limits and temporary blocks too quickly. But still spot broken links in time.

Until that is available, the current implemented way should be ok IMHO.

As you mentioned, the version updater is not perfect yet and might see changes.
But waiting longer on this useful link check - even if the CI setup probably changes - sounds not right. Once the above mentioned feature is available, we do not need the manual "bot call" for link checks anyway.